shell/plugins/dashboard-store/resource-class.js

```diff
@@ -744,7 +744,31 @@ export default class Resource {
   get stateDescription() {
     const trans = this.stateObj?.transitioning || false;
     const error = this.stateObj?.error || false;
-    const message = this.stateObj?.message;
+    let message = this.stateObj?.message;
```

Comment:
do not edit existing lines, comment out and create a new line with your changes.  Made sure both are inside the Added By Verrazzano markers...

Reply:
Thanks, updated.